src/chrome/content/rules/Serving-sys.com.xml

```diff
@@ -1,70 +1,109 @@
+	<target host="centrohyundai.mmgl.cd.serving-sys.com" />
+	<target host="chrysler.mmgl.cd.serving-sys.com" />
+	<target host="collective.mmgl.cd.serving-sys.com" />
+	<target host="gg.mmgl.cd.serving-sys.com" />
+	<target host="greydigital-es.mmgl.cd.serving-sys.com" />
+	<target host="gwa.mmgl.cd.serving-sys.com" />
+	<target host="havas-chi.mmgl.cd.serving-sys.com" />
+	<target host="havasdigitalfr.mmgl.cd.serving-sys.com" />
+	<target host="havasfr.mmgl.cd.serving-sys.com" />
+	<target host="ingnl.mmgl.cd.serving-sys.com" />
+	<target host="kbb.mmgl.cd.serving-sys.com" />
+	<target host="lexus.mmgl.cd.serving-sys.com" />
+	<target host="maxpoint.mmgl.cd.serving-sys.com" />
+	<target host="mediavest-la.mmgl.cd.serving-sys.com" />
+	<target host="meijer.mmgl.cd.serving-sys.com" />
+	<target host="mmmfr.mmgl.cd.serving-sys.com" />
+	<target host="msnpmg-gpu.mmgl.cd.serving-sys.com" />
+	<target host="nissan.mmgl.cd.serving-sys.com" />
+	<target host="novus.mmgl.cd.serving-sys.com" />
+	<target host="pervorm-nl.mmgl.cd.serving-sys.com" />
+	<target host="pnc-bank.mmgl.cd.serving-sys.com" />
+	<target host="png.mmgl.cd.serving-sys.com" />
+	<target host="rapp-au.mmgl.cd.serving-sys.com" />
+	<target host="reckittbenckiser.mmgl.cd.serving-sys.com" />
+	<target host="saatchila.mmgl.cd.serving-sys.com" />
+	<target host="spark.mmgl.cd.serving-sys.com" />
+	<target host="starcom-chi.mmgl.cd.serving-sys.com" />
+	<target host="swellshark.mmgl.cd.serving-sys.com" />
+	<target host="target.mmgl.cd.serving-sys.com" />
+	<target host="ticketsnow.mmgl.cd.serving-sys.com" />
+	<target host="voucher.cd.serving-sys.com" />
+	<target host="toysrus.mmgl.cd.serving-sys.com" />
+	<target host="umcd.mmgl.cd.serving-sys.com" />
+	<target host="umsf.mmgl.cd.serving-sys.com" />
+	<target host="vwus.mmgl.cd.serving-sys.com" />
+	<target host="weatherbug.mmgl.cd.serving-sys.com" />
+	<target host="yal.mmgl.cd.serving-sys.com" />
+	<target host="ds-cc.serving-sys.com" />
+	<target host="datahub.serving-sys.com" />
+	<target host="dh.serving-sys.com" />
+	<target host="ds-vn.serving-sys.com" />
+	<target host="gizmo.serving-sys.com" />
+	<target host="gizmo-s.serving-sys.com" />
+	<target host="origin-gizmo.serving-sys.com" />
+	<target host="origin.partners-demo.serving-sys.com" />
+	<target host="bs.preview.serving-sys.com" />
 	<target host="secure-ds.serving-sys.com" />
-
-	<!--	Special cases:
-				-->
-	<target host="activity.serving-sys.com" />
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.serving-sys\.com$" name="^(ActivityInfo2|eyeblaster|u2)$" /-->
-	<!--securecookie host="^bs\.serving-sys\.com$" name="(?:S_\d+|r1)$" /-->
+	<target host="secure-msntest.serving-sys.com" />
+	<target host="services.serving-sys.com" />
 
 	<securecookie host="^bs\.serving-sys\.com$" name=".+" />
 
 
-	<rule from="^http://activity\.serving-sys\.com/"
+	<!--<rule from="^http://activity\.serving-sys\.com/"
 		to="https://bs.serving-sys.com/" />
```

Comment:
Do we have any issues with this redirect?

Reply:
I commented it out to see if the url test works with a simple http to https redirect without changing the subdomain and left it this way because the check was happy.

Reply:
But there is no redirect for `activity.serving-sys.com` because you also removed the target.

Reply:
indeed, it is back now